packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -76,6 +81,8 @@ class DiffPaneController extends DisposableController {
     core._selectedSnapshotIndex.value = newElementIndex;
     _isTakingSnapshot.value = false;
     derived._updateValues();
+
+    if (newElementIndex == 1) sidePanelController.toggleVisibility(true);
```

Comment:
I don't know that we want the side panel to come in for every first snapshot that is taken. This seems like the sort of thing that would nag users.